tests/foreman/longrun/test_oscap.py

```diff
@@ -300,6 +318,134 @@ def test_positive_push_updated_content(self):
                 vm.execute_foreman_scap_client()
                 self.assertTrue(self.oscapreports.search(host))
 
+    @upgrade
+    @run_only_on('sat')
+    @tier4
+    def test_positive_oscap_run_with_tailoring_file_and_capsule(self):
+        """ End-to-End Oscap run with tailoring files and default capsule
+
+        :id: 346946ad-4f62-400e-9390-81817006048c
+
+        :setup: scap content, scap policy, tailoring file, host group
+
+        :steps:
+
+            1. Create a valid scap content
+            2. Upload a valid tailoring file
+            3. Create a scap policy
+            4. Associate scap content with it's tailoring file
+            5. Associate the policy with a hostgroup
+            6. Provision a host using the hostgroup
+            7. Puppet should configure and fetch the scap content
+               and tailoring file
+
+        :CaseAutomation: notautomated
+
+        :expectedresults: ARF report should be sent to satellite reflecting
+                         the changes done via tailoring files
+
+        :CaseImportance: Critical
+        """
+        if settings.rhel7_repo is None:
+            self.skipTest('Missing configuration for rhel7_repo')
+        rhel7_repo = settings.rhel7_repo
+        rhel7_content = OSCAP_DEFAULT_CONTENT['rhel7_content']
+        hgrp7_name = gen_string('alpha')
+        policy_values = {
+            'content': rhel7_content,
+            'hgrp': hgrp7_name,
+            'policy': gen_string('alpha'),
+        }
+        vm_values = {
+            'distro': DISTRO_RHEL7,
+            'hgrp': hgrp7_name,
+            'rhel_repo': rhel7_repo,
+        }
+        tailoringfile_name = gen_string('alpha')
+        file_path = settings.oscap.tailoring_path
+        tailoring_path = get_data_file(file_path)
+        with Session(self) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            # Creates oscap content for rhel7
+            session.nav.go_to_oscap_content()
```

Comment:
line should be removed
oscap_content.update will do the same when searching for the name and will do navigate_to_entity() before